protos/flyteidl/core/identifier.proto

```diff
@@ -19,7 +19,7 @@ enum ResourceType {
 // Encapsulation of fields that uniquely identifies a Flyte resource.
 message Identifier {
     // Identifies the specific type of resource that this identifier corresponds to.
-    ResourceType resource_type  = 1;
+    core.ResourceType resource_type  = 1;
```

Comment:
@evalsocket mind taking a look at this? do you have any ideas why this generates bad code?